initfiles/componentfiles/thor/run_thor

```diff
@@ -35,7 +34,7 @@ echo $$ > $RUN_THOR_PID_NAME
 export SENTINEL="thor.sentinel"
 while [ 1 ]; do
     # update slaves file in case state of environment has been altered since last run
-    daliadmin $DALISERVER dfsgroup ${groupName} slaves
+    daliadmin server=$DALISERVER dfsgroup ${groupName} slaves
```

Comment:
Not strictly necessary, 1st arg will be treated as server if server= not specified.
Change ok, just curious if other reason you changed?